lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -328,11 +327,19 @@ static int pdu_session_resource_release_t3592_handler(
     /* Abort the registration procedure */
```

Comment:
Please correct the comment.  (Its pdu session procedure)

Reply:
comment addressed.